src/components/TransactionHistoryRows/TransactionDetails/Address.tsx

```diff
@@ -2,19 +2,19 @@ import { Button, Link, useColorModeValue } from '@chakra-ui/react'
 import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
 
 export const Address = ({
-  explorerTxLink,
+  explorerAddressLink,
   address,
   ens
 }: {
-  explorerTxLink: string
+  explorerAddressLink: string
```

Comment:
I'd name this `explorerAddressUri` here - technically from the point of view of unchained/lib, it is indeed a link (you can click it by itself and it brings you to the explorer), but when used to reconstitute a URL, it is effectively an URI.